### ISSUE TITLE
Getting rid of "Trouble" CF failure AND setting entry chimes

### ISSUE THREAD

**Issue by mixedupone** [flarn2006/BHSTools#8]
After getting access and working on this for a while we’ve had success.  But after all we’re left with two problems. 

1. We have a CF trouble code and a constant beeping tone; help? Could it be due to the bus pirate being connected?

2. We have not been able to set up entry/exit chimes when doors or windows are opened; thoughts?

Thank you. And this has been a great process.

**Comment by vermagit** [comment]
I have reported a similar issue in issue #5. The difference is I still have "CH" failure while you seem to report CF.
Also, my constant beeping happens every 24 hours. So it needs to be "canceled" every 24 hours. If we're away beyond that time period, it will keep on beeping until someone "cancels" it. Are you reporting the same issue?
Any solution?

**Comment by mixedupone** [comment]
Yes we have the continuing chirping sound that has been annoying every 24 hours.  I’d love to solve this. Any thoughts as we’re back to square one and unable to use the system

**Comment by vermagit** [comment]
Haha, turned off here too by default unless we're out for extended times.

There are 2 options at this point:
1. The genius @flarn2006 helps us :) OR
2. Take the nuclear option - clean up and set everything up from scratch. I'm unsure what new impediments we'll encounter there and so I'm hesitant
